docs/pages/enterprise/workflow/ssh-approval-jira-cloud.mdx

```diff
@@ -82,19 +82,34 @@ In the webhook settings page, make sure that the webhook will only send Issue Up
 
 We recommend installing Teleport plugins alongside the Teleport Proxy. This is an ideal
 location as plugins have a low memory footprint, and will require both public internet access
-and Teleport Auth access. We currently only provide linux-amd64 binaries, you can also
-compile these plugins from [source](https://github.com/gravitational/teleport-plugins/tree/master/access/jira).
+and Teleport Auth access. 
 
-```code
-$ wget https://get.gravitational.com/teleport-access-jira-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
-$ tar -xzf teleport-access-jira-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
-$ cd teleport-access-jira/
-$ ./install
-$ which teleport-jira
-# /usr/local/bin/teleport-jira
-```
-
-Run `./install` in from 'teleport-jira' or place the executable in the appropriate `/usr/bin` or `/usr/local/bin` on the server installation.
+<Tabs>
+<TabItem label="Download">
+  ```code
+  $ curl -L https://get.gravitational.com/teleport-access-jira-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
+  $ tar -xzf teleport-access-jira-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
+  $ cd teleport-access-jira
+  $ ./install
+  ```
+</TabItem>
+<TabItem label="From Source">
+  To install from source you need `git` and [`go`](https://go.dev/) installed.
```

Comment:
I think we should remove the link from "go" and be have an explicit link to the Go downloads page, e.g., `If you do not have Go installed, visit the Go [downloads page](<link>)`.